plugins/module_utils/network/vyos/argspec/firewall_rules/firewall_rules.py

```diff
@@ -147,9 +157,23 @@ def __init__(self, **kwargs):
                                     },
                                     "type": "dict",
                                 },
+                                "inbound_interface": {
+                                    "options": {
+                                        "group": {
+                                            "type": "str",
+                                        },
+                                        "name": {
+                                            "type": "str",
+                                        },
+                                    },
+                                    "type": "dict",
+                                },
                                 "ipsec": {
                                     "choices": ["match-ipsec", "match-none"],
```

Comment:
1.5 wants this be more specific and rejects the mentioned choices (Example:
fatal: [vyos1.5]: FAILED! => {"changed": false, "msg": "value of ipsec must be one of: match-ipsec, match-none, got: match-ipsec-in found in config -> rule_sets -> rules"})
1.3 and 1.4 are fine. I need to review use case to understand the issue

Reply:
Good find. I recommend we create a forward-looking issue for this as this change will need some version checking (both for testing and the code) and I'd really like to see about refactoring and centralizing the version check and mocking.

Reply:
Interesting that the documentation doesn't show this change yet for 1.5.